Product/SoapUI_Test/RegressionSuite/Passthru/AuditLogging-Passthrough/QueryDocumentAuditLog.properties

```diff
@@ -26,7 +25,6 @@ ActiveParticipant.RoleIDCode.@displayName=Human Requestor
 <!--Source-->
 	
 ActiveParticipant_Source.@UserID=http://www.w3.org/2005/08/addressing/anonymous
-ActiveParticipant_Source.@AlternativeUserID=19233@192.168.1.2
 ActiveParticipant_Source.@UserName=Karl Skagerberg
 ActiveParticipant_Source.@UserIsRequestor=true
 ActiveParticipant_Source.@NetworkAccessPointID=192.168.1.2
```

Comment:
I have seen different value populated for NetworkAccessPointID for DQ in pass through mode in my local.Is this Blob comparison getting used?